### ISSUE TITLE
Service stops right after running a query

### ISSUE THREAD

**Issue by rmeshksar** [arangodb/arangodb#6131]
## My Environment

* __ArangoDB Version__:       3.3.13

## Steps to reproduce

1.  In the web interface run the following query on any collection. I used airport sample collection.
```
_**_FOR airport IN airports
RETURN airport_**_
```
The web interface has been set to limit the result to 1000.

2.  I can see the result but a couple of seconds after that the service stops and I see this message:
You have been disconnected from the server
The connection to the server has been lost. The server may be under heavy load.

Interestingly if I add Limit 1000 to the query it does not fail.

I tried it on two windows machine and also with docker image and with different collections and all of them behaved the same.

In the windows even log I see the following:

```
Faulting application name: arangod.exe, version: 0.0.0.0, time stamp: 0x5b59f922
Faulting module name: ntdll.dll, version: 10.0.17134.165, time stamp: 0xf4df6dc2
Exception code: 0xc0000005
Fault offset: 0x000000000002108f
Faulting process id: 0xe84
Faulting application start time: 0x01d42dff1ccb1673
Faulting application path: C:\Program Files\ArangoDB3 3.3.13\usr\bin\arangod.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report Id: 03ec4776-58b3-410a-9f70-d773f938edb9
Faulting package full name: 
Faulting package-relative application ID: 
```
And in the dump file I see some error related to the ntdll.dll Access Violation Writing location.

Thanks for your help.

**Comment by jsteemann** [comment]
This is very likely a duplicate of https://github.com/arangodb/arangodb/issues/6076, and will be fixed in 3.3.14.

**Comment by sleto-it** [comment]
Hi,

Just a quick message to inform that v.3.3.14, which contains the fix for this problem, has been released

I am going ahead and closing this issue now. Please comment, if needed, and we will reopen it

Thanks again for reporting this bug,
